prooph/pdo-event-store/1.15/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "copy-from-package": {
+        "scripts/": "%CONFIG_DIR%/scripts/"
+    },
+    "env": {
+        "POSTGRES_DSN": "pgsql:host=127.0.0.1;dbname=event_streams",
```

Comment:
Sorry I missed that earlier, but you should probably prefix the env vars with something specific to your bundle.